addons/point_of_sale/static/src/app/navbar/cash_move_popup/cash_move_popup.js

```diff
@@ -30,14 +30,16 @@ export class CashMovePopup extends Component {
     }
 
     async confirm() {
-        const amount = parseFloat(this.state.amount);
-        const formattedAmount = this.env.utils.formatCurrency(amount);
+        let amount = parseFloat(String(this.state.amount).replace(",", "."));
```

Comment:
You can't do this manual replacement of decimal separator. The `parseFloat` is supposed to know about it because it's based on the user's localization.